src/app/pages/auth/register/register.page.scss

```diff
@@ -1,5 +1,18 @@
 hr {
   background: var(--ion-color-primary);
-  height: 2px;
-  width: 100px;
+  height: 5px;
+  width: 80px;
+  margin: 15px 0 15px 0;
 }
+
+.btn-section {
+  margin-top: 40px;
+  ion-button {
+    height: 45px;
```

Comment:
use padding. If this style is used in many pages please use a class and reuse everywhere